resources/views/api/emails/voucher_history_email_textonly.blade.php

```diff
@@ -17,4 +17,4 @@
 
 For more information please go to <a href="http://www.alexandrarose.org.uk/">www.alexandrarose.org.uk</a>.
 
-<a href="http://www.alexandrarose.org.uk/privacy/">Privacy Policy</a>
+<a href="https://voucher-staging.alexandrarose.org.uk/privacy-policy">Privacy Policy</a>
```

Comment:
That should be added to .env and the config/app.php or make a config/arc.php 

Then load with config('app.arc_market_url')